pkg/lvm/lvm_util.go

```diff
@@ -488,9 +516,13 @@ func ReloadLVMMetadataCache() error {
 
 // ListLVMVolumeGroup invokes `vgs` to list all the available volume
 // groups in the node.
-func ListLVMVolumeGroup() ([]apis.VolumeGroup, error) {
-	if err := ReloadLVMMetadataCache(); err != nil {
-		return nil, err
+//
+//If we will call this function from collector.go for metrics collection, exporter will be set as true otherwise false.
+func ListLVMVolumeGroup(exporter bool) ([]apis.VolumeGroup, error) {
```

Comment:
function param is not related to exporter, but rather if you want to reload the lvmmetad cache or not. rename this to `reloadCache bool` or similar param name.